packages/frontend/app/pods/components/pod/application/off-canvas-container/sidebar/contacts/contact/component.ts

```diff
@@ -32,6 +32,12 @@ export default class SidebarContact extends Component<IArgs> {
 
   @reads('settings.hideOfflineContacts') hideOfflineContacts!: boolean;
 
+  get getIsPinned(){
```

Comment:
a getter doesn't need to have `get` in the name, cause there is already a `get` keyword ;)